packages/api/src/auth/decoders/magicLink.ts

```diff
@@ -1,13 +1,28 @@
-export const magicLink = (token: string) => {
-  if (!process.env.MAGICLINK_PUBLIC) {
-    if (process.env.NODE_ENV === 'development') {
-      console.warn(
-        'MAGICLINK_PUBLIC env var is not set. Be certain to set this value in Production.'
-      )
-    } else {
-      console.error('Magic Auth configuration error.')
-      throw new Error('Magic Auth configuration error.')
-    }
+// Relevant type excerpt from Magiclink's magic-sdk/admin:
+// export interface Claim {
+//   iat: number; // Issued At Timestamp
+//   ext: number; // Expiration Timestamp
+//   iss: string; // Issuer of DID Token
+//   sub: string; // Subject
+//   aud: string; // Audience
+//   nbf: number; // Not Before Timestamp
+//   tid: string; // DID Token ID
+//   add: string; // Encrypted signature of arbitrary data
+// }
+// export type ParsedDIDToken = [string, Claim];
+//
+// This function validates the token, throw an error if validations fails
+// otherwise it will return the decoded token in the form of a ParsedDIDToken
+export const magicLink = async (token: string) => {
+  const { MAGIC_SECRET_API_KEY } = process.env
+  if (!MAGIC_SECRET_API_KEY) {
+    throw new Error('`MAGIC_SECRET_API_KEY` environment variable not set.')
```

Comment:
It would be good to `console.warn` this message too.

Reply:
When I test this I'll check if this error is already getting caught / printed to the console.